### PR TITLE
Fix snap: fix line continuation

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

**Comment by blueyed** [comment]
Hard to test, but `openssl` is likely to wait for input there.

**Comment by justinmk** [comment]
Nice catch! My eyes are buggin' out from staring too long. Hope we find a way to debug `deploy.sh` ... I was hesitant to enable `set -x` in case it might reveal the secret token...

**Comment by blueyed** [comment]
One way might be to replace the token in any output again (https://github.com/awesomeWM/awesome/blob/d52e332b96fd09a2069201827624af69c71bae7b/build-utils/travis-apidoc.sh#L137), but better to avoid `set -x`.

btw: have you considered enabling the repo setting to delete merged head branches automatically (on the first settings page).

File: .travis.yml [hunk 1]
```diff
@@ -158,7 +158,7 @@ jobs:
       env:
         - CLANG_SANITIZER=TSAN
         - *common-job-env
-    - if: false
+    - if: type != pull_request
```
Comment (blueyed) [inline .travis.yml]: btw: maybe this should be restricted to "master"?  (i.e. to not run on release branches; or if useful there then likely needs to use another channel (or something alike))

Comment (justinmk) [reply .travis.yml]: well, the plan is to also push to release channel. In theory, that's decided/controlled by https://github.com/neovim/neovim/blob/5c9063cf2d115b652ccdde6f72afc464b7085a18/ci/snap/deploy.sh . 

Not clear if it works yet.

**Comment by justinmk** [comment]
> btw: have you considered enabling the repo setting to delete merged head branches automatically (on the first settings page).

Did so now. Though I would guess that only works for branches in this repo, not forks.

**Comment by blueyed** [comment]
Waiting for manual review:

> Will need manual review...
Sorry, an error occurred in Snapcraft:
The Store automatic review failed.

> https://travis-ci.org/neovim/neovim/jobs/618447865#L4504

**Comment by blueyed** [comment]
> Did so now. Though I would guess that only works for branches in this repo, not forks.

Cool.  It also works for forks.

**Comment by justinmk** [comment]
> Waiting for manual review:

Ah, I think that means we need to wait for them to do it. 

The build showed up in the https://snapcraft.io/nvim/releases admin page, then I tried to put it in the "edge" channel. But when I "Save", it just reverts. Also https://snapcraft.io/nvim is a 404 currently, though the admin area indicates it should be live.

**Pull request merged**